### PR TITLE
Expose the function to add skip button

### PR DESCRIPTION
**Pull request by pajong**
Expose the function to add skip button so that FreeWheel can use the function to setup a skip button.
JW7-3245

### CODE REVIEW

File: src/js/controller/instream-adapter.js [hunk 1]
```diff
@@ -322,6 +331,7 @@ define([
 
         this.setText = function(text) {
             _view.setAltText(text ? text : '');
+            window.yo = this;
```
Comment (robwalch) [inline src/js/controller/instream-adapter.js]: dude

**Comment by robwalch** [comment]
Vast is able to display a skip button without exposing this method. Do we not know at the time of loading an add if it can be skipped?

File: src/js/controller/instream-adapter.js (hunk 1 above)
Comment (pajong) [inline src/js/controller/instream-adapter.js]: oops my bad

**Comment by pajong** [comment]
We do not call loadItem with FreeWheel - there are things called slots that control the ad play back instead of loading items this way. We just use the adapter to start blocking and adding ad texts to control bar.

**Pull request merged**